utils/collectd/files/collectd.uci

```diff
@@ -188,7 +188,8 @@ config globals 'globals'
 
 #config plugin 'tcpconns'
 #	option enable '0'
-#	list ListeningPort '0'
+#	option 'AllPortsSummary' '0'
+#	option ListeninngPort '0'
```

Comment:
Likely a typo:  "ListeninngPort" 
(it was right before changes...)

Reply:
it's boolean in json. Should it really be "list" here?

Reply:
https://collectd.org/wiki/index.php/Plugin:TCPConns#Single_port

Reply:
Ah, I see the typo. Thanks!

Reply:
Based upstream source code, it is a boolean flag. Should be "option", I think.

And I now notice that also the original spelling was wrong (missing s):
not ListeningPort , 
not ListeninngPort, 
but ListeningPorts

https://github.com/collectd/collectd/blob/collectd-5.12/src/tcpconns.c#L260-L261

Reply:
yeah, it's ListeningPort**s** indeed.